-- run_test.py: invoking following command in '/workspace/ros_ws/src/ariaNode':
 - /opt/ros/jazzy/bin/ament_flake8 --xunit-file /workspace/ros_ws/src/ariaNode/build/ariaNode/test_results/ariaNode/flake8.xunit.xml


2 files checked
No problems found

Checked files:

* ./install/_local_setup_util_ps1.py
* ./install/_local_setup_util_sh.py


-- run_test.py: return code 0
-- run_test.py: verify result file '/workspace/ros_ws/src/ariaNode/build/ariaNode/test_results/ariaNode/flake8.xunit.xml'
